easybuild/easyblocks/generic/perlbundle.py

```diff
@@ -82,13 +82,13 @@ def sanity_check_step(self, *args, **kwargs):
                 'dirs': [os.path.join('lib', 'perl%s' % majver)],
             }
 
-        super(Bundle, self).sanity_check_step(*args, **kwargs)
+        super().sanity_check_step(*args, **kwargs)
 
     def make_module_extra(self):
         """Extra module entries for Perl bundles."""
         majver = get_major_perl_version()
         sitelibsuffix = get_site_suffix('sitelib')
 
-        txt = super(Bundle, self).make_module_extra()
+        txt = super().make_module_extra()
```

Comment:
This is not equivalent, `super(Bundle)` jumps into `EasyBlock`, while `super()` jumps into `Bundle`.

Reply:
Similar to above: This ensures the Bundle-specific easyconfig parameters are handled. In this case `altroot/altversion`. Currently a no-op as PerlBundle doesn't use those but might avoid confusion in the future.